spec/factories/school.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+
+  factory :school do
+    email "school@example.com"
```

Comment:
So this is where the Faker gem would be helpful, you can create a fake email, password, address, etc. with it